CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+## Contributing
+Contributions of more lockfree data structures are appreciated! Please create issues/PRs to this repo.
+
+### Maintainers
+The current list of maintainers is as follows:
+
+-   @bartoszmodelski Bartosz Modelski
+-   @kayceesrk KC Sivaramakrishnan
+-   @lyrm Carine Morel
+-   @Sudha247 Sudha Parimala
+
+### Guidelines  for new data structures implementation
+Reviewing a lock free implementation takes times. Here are a few guidelines to make it easier for the reviewers :
+- choose a needed data structure (listed in issues)
```

Comment:
I'd rephrase this as - "The issue tracker has a good list of data structures to choose from" - it could be the case that the contributor has uses of their own not captured in our issue tracker.